src/mscorlib/shared/System/IO/PathInternal.Unix.cs

```diff
@@ -55,23 +62,29 @@ internal static string NormalizeDirectorySeparators(string path)
                 }
             }
 
-            if (normalized) return path;
+            if (normalized)
```

Comment:
why does this return null if normalized, and not just path?

if it does not reutrn null then NormalizeDirectorySeparators need not check for null.

Reply:
Ah, path is a span...